scripts/firedrake-zenodo

```diff
@@ -97,6 +93,13 @@ else:
                         format='%(message)s')
 log = logging.getLogger()
 
+cwd = os.getcwd()
+try:
+    src = os.environ["VIRTUAL_ENV"] + "/src"
```

Comment:
I don't know how to find the source code for a global installation, which makes firedrake-zenodo rather hard to do.